recipe/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D BUILD_SHARED_LIBS=ON ^
+      -D ENABLE_TESTS=OFF ^
+      -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+      -D ZLIB_LIBRARY=%LIBRARY_LIB%\zlib.lib ^
```

Comment:
Nope. .lib can be either. Only way to know for sure is to use tools on the  lib. You can't directly link against a dll at build time.